book/src/user/docker.md

```diff
@@ -54,10 +52,34 @@ docker build \
   .
 ```
 
+### Alternatives
+
+See [Building Zebra](https://github.com/ZcashFoundation/zebra#building-zebra) for more information.
+
+## Advanced usage
+
+You're able to specify various parameters when building or launching the Docker image, which are meant to be used by developers and CI pipelines. For example, specifying the Network where Zebra will run (Mainnet, Testnet, etc), or enabling features like metrics with Prometheus.
+
+### Building with Custom Features
+
+Zebra supports various features that can be enabled during build time using the `FEATURES` build argument:
+
+For example, if we'd like to enable metrics on the image, we'd build it using the following `build-arg`:
```

Comment:
This will become outdated once we enable the `prometheus` by default.

Reply:
I can't think of another full example right now. If this is a blocker, I can prepare an example later. If not, we can fix it with the PR that enables prometheus by default, as a lot of documentation and files (including the docker-compose) will require update with that change.